public/index.html

```diff
@@ -4,57 +4,57 @@
     <meta charset="UTF-8">
     <link href="https://fonts.googleapis.com/css?family=Karla:400,700" rel="stylesheet">
     <link href="https://d3mlfyygrfdi2i.cloudfront.net/favicon.png?v=2" rel="icon" type="image/png">
-    <link rel="stylesheet" href="/styles.css">
-    <link rel="stylesheet" href="http://www.samgetlan.com/navbar/styles.css">
+    <!-- <link rel="stylesheet" href="http://www.samgetlan.com/navbar/styles.css">
     <link rel="stylesheet" href="http://s3-us-west-1.amazonaws.com/fec-kickstarter-campaign-module/styles.css">
     <link rel="stylesheet" href="https://s3-us-west-1.amazonaws.com/updates-and-comments/Updates/styles.css">
-    <link rel="stylesheet" href="http://www.samgetlan.com/community/styles.css">
-    <title>NotAirBnB</title>
```

Comment:
I would recommend taking all of these links and removing them as they should never be relevant again